pkg/plugin/test-keeper/plugin/configuration.go

```diff
@@ -0,0 +1,31 @@
+package plugin
+
+import (
+	"github.com/arquillian/ike-prow-plugins/pkg/plugin/config"
+	"github.com/arquillian/ike-prow-plugins/pkg/scm"
+	"github.com/sirupsen/logrus"
+)
+
+// TestKeeperConfiguration defines inclusion and exclusion patterns set of files will be matched against
+// It's unmarshaled from test-keeper.yml configuration file
+type TestKeeperConfiguration struct {
+	Inclusion      string `yaml:"test_pattern,omitempty"`
+	Exclusion      string `yaml:"skip_validation_for,omitempty"`
+	Combine        bool   `yaml:"combine_defaults,omitempty"`
+	CommentMsgFile string `yaml:"comment_message_file,omitempty"`
+}
+
+// LoadTestKeeperConfig loads a TestKeeperConfiguration for the given change
+func LoadTestKeeperConfig(log *logrus.Entry, change scm.RepositoryChange) (urlIfExists string, conf TestKeeperConfiguration) {
+	configLoader := config.NewPluginConfigLoader(ProwPluginName, change)
+
+	configuration := TestKeeperConfiguration{Combine: true}
+	exists, err := configLoader.Load(&configuration)
+	if err != nil {
+		log.Warnf("Config file was not loaded. Cause: %", err)
+	}
+	if exists {
+		return configLoader.CreateConfigFileURL(), configuration
+	}
+	return "", configuration
```

Comment:
Why are we returning a tuple here? How about having a new field in the struct itself encapsulating it? Then we would probably don't need `exists` as a returned information too (as I guess `err` should be enough to determine that in general).

Reply:
funny, this was one of my thoughts as well :-D. I decided to use the possibility of returning more than one variables to keep the configuration struct "clean" in the sense that it contains only configurable params... But I'm open to adding it there ;-)

Reply:
3aaf76e